### DISCUSSION TITLE
Lack of internet

### DISCUSSION THREAD

**Discussion by sysoev-dev** [AndresGarciaSobrado91/MSI-Modern15-Hackintosh#24] [Q&A]
According to the Open Core instructions, I created a bootable USB flash drive and added your EFI folder and successfully started from the USB flash drive, but I have no Internet and Wi-Fi. As I understand at this stage, I have no way to install HeliPort.dmg. I also tried to connect the phone as a HoRNDIS modem, but that doesn't work either. Any idea how to get internet for online installer?

**Comment by flvoicu** [comment]
You need a usb with a full macos (acces to one macos divice needed) or, you can buy an ethernet to usb adaptor (worked for me).

**Comment by sysoev-dev** [comment]
Do I need a full image of macOS Monterey v12.2 calibrated to usb? Did I understand correctly?

I saw that the author wrote about the offline image 11.2.3 or earlier. And what will be the outcome? Install an old offline image and then just update it?

**Comment by sysoev-dev** [comment]
In general, I downloaded the BDU image for Windows and so I managed to do an offline installation. Thanks to the author for the work!
